src/components/FormItem.vue

```diff
@@ -11,22 +11,33 @@
                 v-bind="getCommonProperties()"
             />
 
+            <v-text-field
+                v-else-if="field.widget.type === 'password'"
+                :type="show_password ? 'text' : 'password'"
+                :append-icon="show_password ? 'visibility' : 'visibility_off'"
+                v-bind="getCommonProperties()"
+                @click:append="show_password = !show_password"
+            />
+
             <v-checkbox
                 v-else-if="field.widget.type === 'checkbox'"
                 v-bind="getCommonProperties()"
             />
             <div v-else>
                 Input item of {{ field.widget.type }} type is not supported yet.
+                {{ printError(field.widget) }}
```

Comment:
Now it seems that we have a duplicity with two almost identical messages. Could we just modify `printError` to return the message and delete the message itself from this place? The same applies to the other use of `printError`.

Reply:
Fixed